avm/res/container-service/managed-cluster/main.bicep

```diff
@@ -753,6 +759,15 @@ resource managedCluster 'Microsoft.ContainerService/managedClusters@2024-03-02-p
   }
 }
 
+module managedCluster_maintenanceConfigurations 'maintenance-configurations/main.bicep' = if (automatic) {
+  name: '${uniqueString(deployment().name, location)}-ManagedCluster-MaintenanceConfigurations'
+  params: {
+    name: 'aksManagedAutoUpgradeSchedule'
```

Comment:
Shoud this be hardcoded (e.g., because it must be a certain value). If so, I'd recommend to move it to the child as a default and remove the param here.